.circleci/config.yml

```diff
@@ -4,16 +4,23 @@ runtest: &runtest
     - run:
         name: Install GPflow
         command: |
-          pip install -U pip
-          pip install -e .
-          pip install -r tests_requirements.txt
+          # Run in a fresh virtual environment, to avoid conflicts with preinstalled packages.
+          virtualenv -p python3.6 .venv
```

Comment:
Does CircleCI always provide a fresh directory? (I remember Jenkins having leftover stuff from previous runs)
(and if it doesn't, what happens if .venv already exists?)